es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number}${street}`;
+
+    console.log('LearningFuze:', address);
+
+    const firstName = 'Uzair';
+    const lastName = 'Ashraf';
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit}${item}`;
+
+    console.log(`I have a ${item}`);
+    console.log(`I have an ${fruit}`);
```

Comment:
This wasn't a concatenation:

---

![image](https://user-images.githubusercontent.com/16143771/127087293-29794861-f135-4871-a053-b94976bec521.png)

---

![image](https://user-images.githubusercontent.com/16143771/127087349-d6f8bfa7-7c6e-4e6c-901f-0b000646ab86.png)